Detectors/FIT/simulation/src/Detector.cxx

```diff
@@ -292,7 +281,7 @@ Bool_t Detector::ProcessHits(FairVolume* v)
     fMC->TrackPosition(x, y, z);
     fMC->CurrentVolID(quadrant);
     fMC->CurrentVolOffID(1, mcp);
-    float time = fMC->TrackTime() * 1.0e12;
+    float time = fMC->TrackTime() * 1.0e9;
```

Comment:
I think it would be good explaining the factor 1E-09 in a comment: `// We are converting from xxx to yyy`